openmc/deplete/chain.py

```diff
@@ -531,6 +531,16 @@ def form_matrix(self, rates, fission_yields=None):
                         if target is not None and path_rate != 0.0:
                             k = self.nuclide_dict[target]
                             matrix[k, i] += path_rate * br
+
+                        # Determine light nuclide production, e.g., (n,d) should
+                        # produce H2
+                        light_nucs = _SECONDARY_PARTICLES.get(r_type)
```

Comment:
Minor comment. Would `_SECONDARY_PARTICLES.get(r_type, [])` work here as well as in `Chain.reduce`? Would remove the conditional check

Reply:
Definitely, thanks for the suggestion!